src/sonic_ax_impl/mibs/ieee802_1ab.py

```diff
@@ -306,10 +306,16 @@ def reinit_data(self):
             logger.debug("Got mgmt ip from db : {}".format(self.mgmt_ip_str))
         try:
             addr_subtype_sub_oid = 4
-            mgmt_ip_sub_oid = (addr_subtype_sub_oid, *[int(i) for i in self.mgmt_ip_str.split('.')])
+            mgmt_ip_sub_oid = None
+            for mgmt_ip in self.mgmt_ip_str.split(','):
+                if '.' in mgmt_ip:
+                    mgmt_ip_sub_oid = (addr_subtype_sub_oid, *[int(i) for i in mgmt_ip.split('.')])
+                    break
```

Comment:
>break [](start = 20, length = 5)

I have a question about design: if there is multiple ipv4 IP addresses for one management port, should we get all of them and exposed in SNMP?

Reply:
To check CLI management description at http://vincentbernat.github.io/lldpd/usage.html, management ip is for system, the first IPv4 and the first IPv6 are used. Please correct me if there is any mistake or error. Thanks.

Reply:
Thanks for the reference! Then we're good.